src/Resources/translations/messages.en.yaml

```diff
@@ -1,5 +1,6 @@
 araise_table:
     download:
+        tooltop: Export
```

Comment:
There is a typo. It should be `tooltip` instead of `tooltop`.

Reply:
Oh, that's not supposed to be here. The correct translation is in the other bundle